etc/spack/defaults/packages.yaml

```diff
@@ -52,3 +52,9 @@ packages:
     permissions:
       read: world
       write: user
+  openmpi:
+    buildable: False
+    externals:
+    - spec: 'openmpi@3.1.6-gcc_9.3.0'
+      modules:
+      - 'openmpi/3.1.6-gcc_9.3.0'
```

Comment:
I assume these changes are unintentional.

We recommend that users don't edit `defaults/packages.yaml` but instead use one of the many other configuration scopes.